Run tests:
    ./py.test

Update Linux USB IDs database
    python manage.py --update-usb-ids
